sql/explain.go

```diff
@@ -48,14 +48,25 @@ func (p *planner) Explain(n *parser.Explain) (planNode, error) {
 	if err != nil {
 		return nil, err
 	}
+	plan, err = markExplain(plan, mode)
+	if err != nil {
+		return nil, fmt.Errorf("%v: %s", err, n)
```

Comment:
No. I usually use `%v` for err because it will print `<nil>` if `err == nil`. `%s` is the correct verb for printing a parse node.